tests/integration/test_storage_iceberg/test.py

```diff
@@ -1510,9 +1575,19 @@ def execute_spark_query(query: str):
         """
     )
```

Comment:
Try to insert new data here and after that make an assertion. Try to insert data in test after each DDL-query (create or alter)

Reply:
Initial test doesn't check a lot of use cases tbh.